From 1f3752c81ad5d746f90db751425c39c3ed0970ab Mon Sep 17 00:00:00 2001
From: David Herrmann <dh.herrmann@gmail.com>
Date: Thu, 11 Sep 2014 15:27:56 +0200
Subject: [PATCH] terminal: remove unused set.h inclusion in idev

We don't use set.h so no need to include it. We used to include it for
temporary refs on all idev devices of a session, but that never was pushed
upstream.
---
 src/libsystemd-terminal/idev.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libsystemd-terminal/idev.c b/src/libsystemd-terminal/idev.c
index 0ed518cded..d37e0313e4 100644
--- a/src/libsystemd-terminal/idev.c
+++ b/src/libsystemd-terminal/idev.c
@@ -33,7 +33,6 @@
 #include "idev-internal.h"
 #include "login-shared.h"
 #include "macro.h"
-#include "set.h"
 #include "udev-util.h"
 #include "util.h"
 
